.circleci/config.yml

```diff
@@ -114,6 +114,9 @@ jobs:
       - run:
           name: Test amplify
           command: xcodebuild test -enableThreadSanitizer NO -workspace Amplify.xcworkspace -scheme Amplify -sdk iphonesimulator -destination "${destination}" | xcpretty --simple --color --report junit
+      - run:
+          name: Upload coverage report to Codecov
+          command: bash <(curl -s https://codecov.io/bash) -F $CIRCLE_JOB
```

Comment:
Let's copy the script at https://codecov.io/bash into our build-support directory and invoke it from there. That gives us the opportunity to review it for security before executing it, and protects against cases where the `bash` endpoint is unavailable or the script is broken.